VARIABLES.md

```diff
@@ -246,7 +246,7 @@ Default:  true
 
 ### regenerate_keystore_and_truststore
 
-Boolean to have reruns of all.yml recreate Keystores
+Boolean to have reruns of all.yml recreate Keystores. Consider disabling this once installation is completed, as this triggers restarts.
```

Comment:
This file gets written via the python script doc.py. Please find the regenerate_keystore_and_truststore variable definition in the source code, update the annotation above it and run the script
```
python doc.py
```